x/ecocredit/client/tx.go

```diff
@@ -98,10 +98,10 @@ Parameters:
 			}
 
 			msg := ecocredit.MsgCreateClass{
-				Admin:      admin.String(),
-				Issuers:    issuers,
-				Metadata:   b,
-				CreditType: creditType,
+				Admin:          admin.String(),
+				Issuers:        issuers,
+				Metadata:       b,
+				CreditTypeName: creditType,
```

Comment:
Nit: can we rename the variable also to `creditTypeName` name?

Also, I think the usage strings for the CLI should be updated (e.g. this file [lines 50-65](https://github.com/regen-network/regen-ledger/blob/ty/msg_createclass/x/ecocredit/client/tx.go#L50-L65)) to [credit-type-name] as a cli argument instead of credit type.